functable.c

```diff
@@ -206,6 +206,8 @@ static void init_functable(void) {
 #ifdef ARM_CRC32
     if (cf.arm.has_crc32) {
         ft.crc32 = &crc32_armv8;
+        ft.crc32_fold = &crc32_fold_armv8;
```

Comment:
Why are these functions needed? Usually if `crc32_fold` does not exist it will automatically do `memcpy`.